src/parser/mod.rs

```diff
@@ -460,7 +460,7 @@ impl Parser {
 
             loop {
                 self.next_token();
-                exprs.push(self.parse_expression(Precedence::Lowest)?);
+                exprs.push(Some(self.parse_expression(Precedence::Lowest)?));
```

Comment:
oh, cc @Tangruilin https://github.com/shaun-io/shaun-rs/blob/7974f22f211d3f5d7990f6b3011f631ba42a423e/src/parser/stmt.rs#L82-L86
looks `InsertStmt` values should't be `Option`?

Reply:
I haven't hacked into this `InsertStmt `structure,` although it can also remove the `Option`, if possible, I can change it

Reply:
> I haven't hacked into this `InsertStmt `structure,`although it can also remove the`Option`, if possible, I can change it

Emmm, you can open an other issue to trace this question if you want to solve it.  😃

Reply:
> > I haven't hacked into this `InsertStmt `structure,`although it can also remove the`Option`, if possible, I can change it
> 
> Emmm, you can open an other issue to trace this question if you want to solve it. 😃

I just saw it out of the blue and questioned it.